webcompat/form.py

```diff
@@ -203,19 +226,20 @@ def build_formdata(form_object):
         'browser': form_object.get('browser'),
         'os': form_object.get('os'),
         'problem_type': get_problem(form_object.get('problem_category')),
-        'description': form_object.get('description')
+        'browser_test_type': get_browser_test(form_object.get('browser_test')),
```

Comment:
for buildFormData() comments, I'm not sure if these items need to stay in:

    Summary -> title
    Version -> part of body